.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
```

Comment:
- Make sure to properly format the COMMIT_DETAILS variable to avoid any unexpected behavior, especially in multi-line scenarios.